docs/persistentvolume-metrics.md

```diff
@@ -1,11 +1,12 @@
 # PersistentVolume Metrics
 
-| Metric name| Metric type | Labels/tags | Status |
```

Comment:
Is this diff large because we're adding two new columns?

Reply:
Yes